chaos/cases/task.go

```diff
@@ -316,20 +339,53 @@ func (t *task) genIncrData(ctx context.Context) (err error) {
 		}
 	}()
 
+	runCaseSQLs := func() error {
+		testSQLs := t.caseGenerator.GetSQLs()
+		if testSQLs == nil {
+			getNewCase = false
+			return nil
+		}
+		for _, testSQL := range testSQLs {
+			log.L().Info("execute test case sql", zap.String("ddl", testSQL.statement), zap.Int("source", testSQL.source))
+			if err2 := t.sourceConns[testSQL.source].execDDLs(testSQL.statement); err2 != nil {
+				return err2
+			}
+		}
+		return nil
+	}
+
+	defer func() {
+		log.L().Info("complete test case sql")
+		for {
+			if !getNewCase {
+				return
+			}
+
+			if err2 := runCaseSQLs(); err2 != nil {
```

Comment:
we `runCaseSQLs` in below for loop. Here we want to make sure run it at least once?

Reply:
Make sure it runs at least once, and make sure all cases will be completed.